base/changes.txt

```diff
@@ -6,6 +6,10 @@ to completeness or accuracy and it contains some references to files that are
 not part of the distribution.
 ================================================================================
 
+2025-01-22  Joseph Wright  <Joseph.Wright@latex-project.org>
+	* ltcmd.dtx, usrguide.tex
+	New "f"-type argument
```

Comment:
I still don't like "f", it doesn't resonate with me. Have we ruled out "c" for collect, if so why? Could then perhaps one day extended to `C{<setup>}` with some special setup for the collection.

Reply:
It is a 'working title' - I am not wedded to it. I'm quite happy if we prefer `c` or one of the other suggestions. Probably needs a 'vote'!

Reply:
artistic license @Skillmon :-) but yes you are probably right, but that doesn't rule out "c" rather than "f"

Reply:
@Skillmon  In the end it's a guideline of course - we could decide that the utility trumps the consistency argument. But I wonder if instead we might want some more generic catcode setup - would apply to `v` as well, e.g. `*{<catcodes>}`?

Reply:
In my opinion, letter `c` ('code') is better than `f` ('filecontents').